AppCenter/AppCenter/Internals/Channel/MSACChannelUnitDefault.h

```diff
@@ -89,6 +89,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic) BOOL discardLogs;
 
+/**
+ * Flag indicating whether SDK network requests are allowed.
+ */
+@property(nonatomic, getter=isNetworkRequestsAllowed, setter=setNetworkRequestsAllowed:) BOOL networkRequestsAllowed;
```

Comment:
Do we need `setNetworkRequestsAllowed` here? properties `enabled` and `paused` in this file don't have such.